test/malli/instrument/cljs_test.cljs

```diff
@@ -47,6 +48,23 @@
 
     (is (= 4 (power "2")))
     (is (= 36 (power 6))))
+  (testing "without instrumentation, from another ns"
```

Comment:
Can you move the unstrument after the instrument in the test, this will actually test if the fns are unstrumented. (There was a bug with  the filters in the past so it's a good idea to put them in that order.)